repo/packages/O/openvpn/0/marathon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "{{openvpn.framework-name}}",
+  "cpus": 0.01,
+  "mem": 128,
```

Comment:
Thanks, good catch. Fixed! Will submit another PR soon to include a small linter for duplicated keys.